### PR TITLE
Navigation menu - Update

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@codiebeulaine @KaitCrawford 
Please review | Position section on part of the menu

Screenshot 2019-05-14 at 14 09 19
Screenshot 2019-05-14 at 14 09 33

File: gem/templates/rwanda/base.html [hunk 1]
```diff
@@ -141,11 +141,10 @@
...
                   {% for section_cat in sections %}
-                    {% get_translation section_cat as section %}
-                    {% if section.title == 'Ibibera Ino' or section.title == 'Events' %}
-                      <a href="{% pageurl section.specific %}" class="nav-list__anchor">
-                       {{section.title}}
-                      </a>
+                    {# {% get_translation section_cat as section %} #}
+                    {% content_is section_cat "Events" as is_event %}
+                    {% if is_event == True %}
+                      <a href="{% pageurl section_cat.specific %}" class="nav-list__anchor">{{section_cat.title}}</a>
```
Comment (KaitCrawford) [inline gem/templates/rwanda/base.html]: Does this not need to be translated somewhere in this? It looks like you put the `get_translation` tag above here into a comment.

File: gem/templates/rwanda/base.html [hunk 2]
```diff
@@ -175,12 +174,15 @@
                 <h1 class="nav-list__title">{% trans "Featured" %}:</h1>
                 <ul class="nav-list-inner">
                   {% for section_page in sections %}
-                    {% if section_page.show_in_menus == True %}
-                      <li class="nav-list-inner__item">
-                        <a href="{% pageurl section_page.specific %}" class="nav-list__anchor">
-                          {{section_page.title}}
-                        </a>
-                      </li>
+                    {% content_is section_page "Events" as is_event %}
```
Comment (KaitCrawford) [inline gem/templates/rwanda/base.html]: Does this handle translations? I presume the `sections` list that we're looping through is the translated sections.

File: gem/templates/rwanda/base.html [hunk 3]
```diff
@@ -189,12 +191,15 @@ <h1 class="nav-list__title">{% trans "Featured" %}:</h1>
                 <h1 class="nav-list__title">{% trans "News and stories" %}:</h1>
                 <ul class="nav-list-inner">
                   {% for section in sections %}
+                    {% content_is section "Events" as is_event %}
```
Comment (KaitCrawford) [inline gem/templates/rwanda/base.html]: Same question about translations here

File: gem/templates/rwanda/base.html [hunk 4]
```diff
@@ -266,22 +271,22 @@ <h3 class=" heading__basic">
...
               {% if sections %}
                 {% for section in sections|slice:":4" %}
-                  {% pageurl section.specific as section_path %}
-                    {% if section_path != '/sections/events/' %}
-                      <li class="menu-list__item">
-                        <a href="{% pageurl section.specific %}" class="menu-list__anchor {% if request.path == section_path %}menu-list__anchor--selected{% endif %}">
-                          {{section.title}}
-                        </a>
-                      </li>
+                  {% content_is section "Events" as is_event %}
```
Comment (KaitCrawford) [inline gem/templates/rwanda/base.html]: Same question as above about handling translations.

File: gem/templates/rwanda/base.html [hunk 5]
```diff
@@ -266,22 +271,22 @@ <h3 class=" heading__basic">
...
                   {% endif %}
                 {% endfor %}
               {% endif %}
             </ul>
             <ul class="menu-list menu-list--column2">
               {% if sections %}
                 {% for section in sections|slice:"4:" %}
-                  {% pageurl section.specific as section_path %}
-                  {% if section_path != '/sections/events/' %}
+                  {% content_is section "Events" as is_event %}
```
Comment (KaitCrawford) [inline gem/templates/rwanda/base.html]: Translations?

**Comment by Mitso** [comment]
@KaitCrawford The _{% content_is ...%}_ tag checks for a keyword (Provided in  English) match / existence  against all translated languages by True or False. I need this to separate a section or place it on a different part of a page ie. Events section from the returned   _{% for section_cat in sections %}_ loop of section indexes on the general navigation menu.


{# {% get_translation section_cat as section %} #} Is commented out until I've tested and made sure there are no translations side-effects in QA.

**Pull request merged**